k9mail/k-9
# Add font size setting for account name in message view

Because there's a minimum size for the chip, reducing the font size won't significantly reduce the space occupied by the chip. A follow-up PR will change that.

To test:
1. Make sure there's more than one account set up.
2. Change font size under *Settings → General settings → Display → Font size → (Messages) → Account name*.
3. Open a message from the Unified Inbox (or any other message list that includes messages from more than one account).
4. Check colored chip with the account name at the top of the screen.
